server.js

```diff
@@ -27,6 +30,51 @@ app.get('/', function (req, res) {
      res.sendFile(__dirname + '/src/index.html');
 });
 
+//Marketing endpoint
+app.get('/marketing', function(req,res){
+    const con = mysql.createConnection(databaseConnectionOptions);
+    var today= moment.utc();
+    //Starting date of this week
+    var thisWeek = today.startOf("week");
+    //date from 2 weeks ago
+    var twoWeeks = thisWeek.clone().subtract(2,"week");
+    //this month start date
+    var thisMonth = today.startOf("month");
+    //last month start date
+    var lastMonth = thisMonth.clone().subtract(1, "month");
+    // //Get all registrations
+    var query = "SELECT COUNT(*) as total FROM str_register; ";
+    query+= `SELECT COUNT(*) as total FROM str_register WHERE created_at >= '${thisWeek.format("YYYY-MM-DD hh:mm:ss")}';`;
```

Comment:
Try not to repeat yourself when you can. put this "YYYY-MM-DD hh:mm:ss" in a variable and use it. This way if for some reason we needed to change that format we can change it from one place